cppcache/src/LocalRegion.cpp

```diff
@@ -367,11 +372,27 @@ void LocalRegion::put(const std::shared_ptr<CacheableKey>& key,
   throwExceptionIfError("Region::put", err);
 }
 
+std::shared_ptr<Serializable> LocalRegion::putIfAbsent(
+    const std::shared_ptr<CacheableKey>& key,
+    const std::shared_ptr<Serializable>& value,
+    const std::shared_ptr<Serializable>& cbArg) {
+  auto opTime = startStatOpTime();
+  std::shared_ptr<Serializable> retValue;
+  std::shared_ptr<VersionTag> versionTag;
+  GfErrType err = putIfAbsentImpl(key, value, cbArg, retValue, -1,
```

Comment:
Is the Impl suffix a common convention in C++ to name methods that implement the logic of public methods?
Given that it is a convention in Java to name classes that implement an interface, I would change the name to something else to avoid misunderstandings.